cli/src/pcluster/constants.py

```diff
@@ -237,6 +237,7 @@ class Feature(Enum):
     SLURM_DATABASE = "SLURM Database"
     CLUSTER_HEALTH_METRICS = "Cluster Health Metrics"
     CAPACITY_BLOCK = "Capacity Block"
+    NLB_SECURITY_GRROUP = "NLB Security Group"
```

Comment:
1. typo: GRROUP => GROUP
1. The string is customer facing, so I suggest to expand the acronym NLB to "Network Load Balancer"

Reply:
Done